src/test/java/com/beust/jcommander/JCommanderTest.java

```diff
@@ -712,6 +713,42 @@ public void variableArityZeroNonBoolean() {
         new JCommander(va).parse("-variable", "a", "b", "c", "d");
     }
 
+    enum Color {
+        RED, BLUE
+    }
+
+    @Test
+    public void enumConverter(){
+
+
+        class ColorEnumConverter extends EnumConverter<Color> {
+            /**
+             * Constructs a new converter.
+             *
+             * @param optionName the option name for error reporting
+             * @param clazz      the enum class
+             */
+            public ColorEnumConverter(String optionName, Class<Color> clazz) {
+                super(optionName, clazz);
+            }
+
+            @Override
+            public Color convert(String value) {
+                return Color.valueOf(value);
+            }
+        }
+
+        class Arg {
+            @Parameter(names = "-colors", converter = ColorEnumConverter.class)
```

Comment:
Correct it does the idea is to have a working test case with a dummy implementation to see how EnumConverter can be used, and refer to it when an example is needed.

At least in other cases I had the experience that this comes handy sometimes :)